documentation/concepts.md

```diff
@@ -202,3 +261,9 @@ allows publishers to *enhance* messages by providing additional metadata and
 serializing their messages into [*envelopes*](./envelope_protocol.md). An
 envelope allows publishers to set things like the `AckInbox`, `Key`, `Headers`,
 and other pieces of metadata.
+
+```
+Do not underestimate the need for meta data on your processes. Proper enrichment of meta data will enable future business models served as new products to new
+consumers, e.g. consumer behaviour etc.
+```
```

Comment:
```suggestion
> **Architect's Note**
>
> Do not underestimate the need for metadata on your processes. Proper enrichment of metadata will enable future business models served as new products to new consumers, e.g. consumer behavior, etc.
```

Reply:
This is not really relating to Liftbridge, but more to the business model evolving over time. The concept is as follow. You starve you competitors by offering the main product for free. You add enough meta data to the main product to enable surrounding add-on products that they are now willing to pay for, since your product is for free with great extra value added (with some additional cost which is equal to what yo gave up on the main product). The competition is gone and your consumer base is larger providing a better revenue base for you. I feel it is an important feature not only for technical meta data but also for product improvment. Do you agree? If not just delete this comment